File: 릿코드/0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        int nums1Length = nums1.length, length = nums1Length + nums2.length;
        int[] nums = new int[length];
        for(int i=0; i<length; i++) 
            if(i < nums1Length) nums[i] = nums1[i];
            else nums[i] = nums2[i - nums1Length];
        
        Arrays.sort(nums);
        
        if(length % 2 == 0) return (nums[length/2-1] + nums[length/2]) / 2.0;
        return nums[length/2];
    }
}

File: 릿코드/0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
const findMedianSortedArrays = (nums1, nums2)  => 
[...nums1, ...nums2]
.sort((a,b) => a-b)
.splice(Math.ceil((nums1.length+nums2.length)/2)-1, (nums1.length+nums2.length) % 2 == 0 ? 2 : 1)
.reduce((x, y) => (x+y)/2).toFixed(5);